process/modules/converb_to_conj.py

```diff
@@ -0,0 +1,180 @@
+from process.module import ParaphraseModule
+from process.preprocessing_utils import PreprocessingUtils
+from stanza.utils.conll import CoNLL
+
+import stanza
+import pymorphy2 
+morph = pymorphy2.MorphAnalyzer()
+import pyconll
+import udon2
```

Comment:
PEP-8:

> Импорты должны быть сгруппированы в следующем порядке:
> - импорты из стандартной библиотеки
> - импорты сторонних библиотек
> - импорты модулей текущего проекта

Две строки после импортов.

MorphAnalyzer лучше создать как атрибут, см. L17.

```suggestion
import pyconll
import pymorphy2 
import stanza
from stanza.utils.conll import CoNLL
import udon2

from process.module import ParaphraseModule
from process.preprocessing_utils import PreprocessingUtils

```